grammars/level5-Additions.lark

```diff
@@ -45,7 +45,7 @@ nospace: /[^\n, ]/
 textwithspaces: /(?:[^#\n،,，、 ]| (?!else|başka|faese|अन्यथा|否则|muuten|muidu|senão|ellers|alie|altfel|иначе|inače|altrimenti|інакше|lainnya|arall|annars|değilse|anders|ndryshe|inaczej|sinon|ไม่อย่างนั้น|sonst|లేకపోతే|sino|ورنہ|وکھرا|אחרת|وإلا))+/ -> text //anything can be parsed except for a newline and a comma for list separators
 //a space is allowed, but it may not be followed by an else. The part " (?!else))" means space not followed by (negative look ahead) else
 //That is because allowing else in strings leads to issue #303
-textwithoutspaces: /(?:[^#\n،,，、 *+\-\/eidіиlలేไamfnsbअ否אو]|א(?!חרת )|و(?!إلا |کھرا |رنہ)|否(?!则 )|अ(?!न्यथा )|и(?!наче )|m(?!uidu |uuten )|d(?!eğilse )|b(?!aşka )|n(?!dryshe )|ไ(?!ม่อย่างนั้น )|l(?!ainnya )|f(?!aese )|e(?!lse |llers )|s(?!inon |enão |ino |onst )|і(?!накше )|i(?!naczej |nače )|లే (?!కపోతే )|a(?!nders |lie |ltfel |ltrimenti |nnars |rall ))+/ -> text //anything can be parsed except for spaces (plus: a newline and a comma for list separators)
+textwithoutspaces: /(?:[^#\n،,，、 *+\-\/eidіиlలేไamfnsbअ否אو]|א(?!חרת )|و(?!إلا |کھرا |رنہ)|否(?!则 )|अ(?!न्यथा )|и(?!наче )|m(?!uidu |uuten )|d(?!eğilse )|b(?!aşka )|n(?!dryshe )|ไ(?!ม่อย่างนั้น )|l(?!ainnya )|f(?!aese )|e(?!lse |llers )|s(?!inon |enão |ino |onst )|і(?!накше )|i(?!naczej |nače )|లే (?!కపోతే )|a(?!nders |lie |ltfel |ltrimenti |nnars |rall ))+/ _END_BLOCK* -> text //anything can be parsed except for spaces (plus: a newline and a comma for list separators)
```

Comment:
I originally started with changing the end block to %ENDBLOCK. However, I quickly realized this will make me escape the % sign from multiple text-like definitions in upper levels (e.g. textwithoutspaces, textwithspaces, text_list and more). And since the text-like definitions are complicated (with negative lookahead that includes translated keywords), I decided to piggyback on a symbol which is already escaped from lower levels. If fact, I came to the realization that by changing the endblock, I need to do way more changes than if I just escape it from the comment.

Also, I need to point out that regardless of whether we change the symbol of the endblock or escape the end block from the comment definition, we need to add an optional ENDBLOCK to the textwithoutspaces definition in level 5 (this is the line you left the comment for). If we do not have this optional ENDBLOCK, then the presence of the error block leaks out in the error message:
<img width="689" alt="Screenshot 2024-04-19 at 13 56 55" src="https://github.com/hedyorg/hedy/assets/3010723/1f6e68fd-9a52-46b1-924f-c7552f67d601">